### PR TITLE
ROECCT-180 modify page individual BO page template query param

### PR DESCRIPTION
**Pull request by ConnorOS-ch**
### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/beneficial-owner-individual.html [hunk 1]
```diff
@@ -4,17 +4,26 @@
...
     <span class="govuk-caption-xl">Beneficial owner</span>
 
     <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
-    <br> <p class="govuk-body">You can add more later.</p> <br>
-
-    {% if entity_number %}
-      {% include "includes/ceased-date-details-summary-new-bo.html" %}
+    {% if relevant_period %}
+      {{ govukInsetText({
+        text: 'You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner.'
```
Comment (cmowatt-ch) [inline views/includes/page-templates/beneficial-owner-individual.html:16]: string enclosed here in single quotes. Changing these to double quotes is more readable and consistent with majority of text: strings in html files. e.g. 
'You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner.'
->
"You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner."

**Review by cmowatt-ch — COMMENTED** [review]
Looks good. Just one comment on the beneficial-owner-individual.html

File: views/includes/page-templates/beneficial-owner-individual.html (hunk 1 above)
Comment (ConnorOS-ch) [reply views/includes/page-templates/beneficial-owner-individual.html]: Thanks, I've updated this :)

**Comment by ndeekonda-ch** [comment]
Looks good to me.

**Review by ndeekonda-ch — APPROVED** [review]

**Pull request merged**